src/store/card.js

```diff
@@ -159,6 +159,20 @@ export default {
 				existingCard.labels = card.labels
 			}
 		},
+		cardIncreaseAttachmentCount(state, cardId) {
+			const existingIndex = state.cards.findIndex(_card => _card.id === cardId)
+			if (existingIndex !== -1) {
+				const existingCard = state.cards.find(_card => _card.id === cardId)
+				existingCard.attachmentCount++
```

Comment:
Is that actually updating instantly then when triggering the mutation. We might need to use Vue.set